_IBC/archway-juno.json

```diff
@@ -2,28 +2,29 @@
   "$schema": "../ibc_data.schema.json",
   "chain_1": {
     "chain_name": "archway",
-    "client_id": "07-tendermint-18",
-    "connection_id": "connection-19"
+    "client_id": "07-tendermint-121",
+    "connection_id": "connection-119"
   },
   "chain_2": {
     "chain_name": "juno",
-    "client_id": "07-tendermint-387",
-    "connection_id": "connection-379"
+    "client_id": "07-tendermint-583",
+    "connection_id": "connection-550"
   },
   "channels": [
     {
       "chain_1": {
-        "channel_id": "channel-15", 
+        "channel_id": "channel-208",
```

Comment:
Please show evidence token balances from the old channel (need to make sure this update isn't abandoning transferred tokens)

Reply:
It looks like there are no ARCH on Juno, but there is a small amount of JUNO on Archway (but small enough that we can proceed)
<img width="1416" height="335" alt="image" src="https://github.com/user-attachments/assets/8660fa63-a02f-412f-9e43-311d586bc279" />